questionnaire/submissions.test.js

```diff
@@ -40,10 +40,34 @@ jest.doMock('../services/slack', () => {
     }));
 });
 
+const messageBusPost = jest.fn((queueName, payload) => {
+    if (
+        [
+            '4ddb0208-f7da-4237-a244-34e7e58d2ddf',
+            '95118e91-8ed9-4b41-a7a8-727a353f2d18',
+            'be567065-d11c-4ba1-8c3f-147290c7036f',
+            '93259f6d-1826-4e97-ba39-53b4e232dd81'
+        ].includes(payload.applicationId)
+    ) {
+        throw new Error('some error');
+    }
+});
+jest.doMock('../services/message-bus/index.js', () => {
+    return jest.fn(() => ({
+        post: messageBusPost
+    }));
+});
```

Comment:
This doesn't match the message bus caller behaviour

Reply:
again.. rubbish tests for a quick and dirty endpoint. this `throw` is actually emulating the `got` module throwing. as far as the submissions-service is concerned, the behaviour is identical. Not 100% ideal... however let's get something out. it effectively does exactly as expected.

Reply:
Either I'm missing something? or this isn't doing what you think it is. `got` is asynchronous and therefore so is the message bus caller. This mock isn't asynchronous. I'm happy to get something out, but it needs to work as expected and I don't think this does.

Reply:
this is just to force an error when a specific id is used. i.e. emulating a failure in a specific scenario. I will adjust it to be async as that was an error